types/fs-plus/index.d.ts

```diff
@@ -0,0 +1,305 @@
+// Type definitions for fs-plus 3.0
+// Project: http://atom.github.io/fs-plus
+// Definitions by: My Self <https://github.com/me>
```

Comment:
It should be your name and github url here.

Reply:
Oops sorry, I created a bunch of type definitions today, and skipped this. I'll fix.